CHANGELOG.unreleased.md

```diff
@@ -17,6 +17,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 
 ### Changed
 - The brush tool is disabled in low magnifications (magnification 16 and lower) to avoid performance problems when annotating. [#5017](https://github.com/scalableminds/webknossos/pull/5017)
+- The fill tool is disabled in low magnifications (magnification 2 and lower) to avoid producing spotty annotations. [#5050](https://github.com/scalableminds/webknossos/pull/5050)
```

Comment:
```suggestion
- The fill tool is disabled in low magnifications (magnification 2 and lower) to avoid producing spotty annotations (caused by needing too much main memory). [#5050](https://github.com/scalableminds/webknossos/pull/5050)
```